src/router/controllers_v1/PlanStatusController.js

```diff
@@ -66,10 +66,14 @@ export default class PlanStatusController {
       }
 
       if (status.code === PLAN_STATUS.WRONGLY_MADE_WITHOUT_EFFECT) {
-        const [prevLegal] = await PlanSnapshot.findSummary(db, {
-          plan_id: planId,
-          is_current_legal_version: true,
-        });
+        // eslint-disable-next-line no-unused-vars
+        const { rows: [standsReview, standsNotReview, prevLegal] } = await db.raw(`
+          SELECT * FROM plan_snapshot 
+          WHERE status_id = ANY(?) 
+          AND plan_id = ?
+          ORDER BY created_at DESC;
+        `, [Plan.legalStatuses, planId]);
```

Comment:
If you join (or just pull from) `plan_snapshot_summary` for rows having an effective legal end date, you'll end up with just the legal versions and also ignore discards.

Reply:
Can do it either way but I'm not sure if you are filtering for discards unless I'm not seeing it

Reply:
Isn't the snapshot with no effective legal end date going to be the stands one for the amendment that was decided wrongly made?

I can also add filtering for discards, that's a good idea.

Reply:
Any legal one, even the current one, will have a legal end date now.  The current one is displaying the plan end date.

The most recent legal one would be the first stands record.  The one previous to that I think is what we would copy.

Reply:
Do you think we can reliably assume the legal version in `plan_snapshot_summary` right before the most current legal one will always be the one we want to copy from? I want it to be resilient to any changes we might have to make to the minor amendment flow (specifically the decision maker statuses we'll be working on soon).

Reply:
I would say so, it's what we are relying on for displaying what is/was legal and from what start to end day.  If that is ever incorrect it would be something we would need to fix.

If we had workflow transaction id's we would look for the last legal one of a different workflow transaction id.  Without it as long as we only let 1 stands count as legal we will be ok.

To prove (mostly to myself thinking it through) that this works repeatably:
1. Let's pretend we have an approved one as v1 (not version 1, but legal version 1 as in it has a legal start and end)
2. We make a minor and it goes to stands and any number of stands statuses, the first of those is our new legal as v2
3. It goes to WMWE, a clone of v1 is made right away (v3) based on max(legal version) - 1 [v1]
4. Now v3 is the most recent.
5. We make another minor, just to prove that this is repeatable, v4 is stands
6. again WMWE, clone of max(legal_version) - 1 [v3]
7. v5 is most recent legal, and looks just like v3, and looks just like v1, which makes sense  because no amendments stuck.

Reply:
Gotcha, I think that makes sense. Since all minor amendments go directly to "stands - not reviewed", I think that's safe to use as a reference point.

Reply:
only thing we didn't capture here is the handling of plan status history and perhaps overwriting it onto the legal copies so that the history on the working plan makes sense.